admin/public/src/containers/Bucket.jsx

```diff
@@ -21,45 +21,53 @@ export default class Bucket extends Component {
   }
 
   render() {
-    const { bucket, handleRemove, showDynamicStats } = this.props
+    const { canMakeChanges = true, bucket, handleRemove, showDynamicStats } = this.props
 
     return (<div className="bucket flex-tile flex-box">
       <div className="flex-container legend">
         <h4>{bucket.name}</h4>
-        <button className="btn btn-danger" onClick={handleRemove}>Remove Bucket</button>
+        {canMakeChanges &&
+          <button className="btn btn-danger" onClick={handleRemove}>Remove Bucket</button>
+        }
       </div>
       <Field keyName="size"
         parent={bucket.name}
+        disabled={canMakeChanges === false}
```

Comment:
Why not `!canMakeChanges`?

Reply:
I want to be sure that the fields are disabled ONLY if this is explicitly set to `false`. Everything else will be considered truthy.

Reply:
fair enough, just figured since you're already doing the conversion to a bool in `NamespaceTile`.